jskit/encoders/utils/train_config.json

```diff
@@ -4,10 +4,10 @@
     "train_batch_size": 8,
     "eval_batch_size": 2,
     "max_history": 4,
-    "learning_rate": 0.01,
-    "weight_decay": 0.1,
-    "warmup_steps": 2000,
-    "adam_epsilon": 8e-08,
+    "learning_rate": 1e-3,
```

Comment:
I wonder if this is too small for an initial learning rate... Do we have any data that looks at this?